main_visual.py

```
@@ -10,6 +10,8 @@
 import pygame
 import numpy as np
 import os
+import math
```

Ruff (0.12.2)

`math` imported but unused

Remove unused import: `math`

(F401)